python3/ltk/actions/action.py

```diff
@@ -198,6 +198,14 @@ def _update_document(self, file_name):
         self.doc_manager.update_document('sys_last_mod', sys_last_modified, doc_id)
         # whenever a document is updated, it should have new translations
         self.doc_manager.update_document('downloaded', [], doc_id)
+        if next_document_id:
+            self.doc_manager.update_document('id', next_document_id, doc_id)
+
+    def update_doc_response_manager(self, response, document_id, file_name, *args, **kwargs):
```

Comment:
This seems to be more like a locked response handler than an update response manager to me, could more functionality be added/ the name changed to reflect its current behavior?

Reply:
Yes, that is good feedback. It was originally going to handle all of the responses but that turned out to be not feasible.